lib/rex/post/meterpreter/ui/console/command_dispatcher.rb

```diff
@@ -50,6 +50,23 @@ def client
   end
 
   #
+  # Returns the commands that meet the requirements
+  #
+  def check_commands(all, reqs=nil)
```

Comment:
Personal preference here would be renaming the function to something like `filter_commands` given that's what it does.